### PR TITLE
Chest upgrades

### PR DESCRIPTION
**Pull request by jsoconnell**
- contains: "does this chest contain this item?"
- get_contents: allows you to get a list of items in the chest w/o looting it
- is_empty: checks if the chest has any items
- is_not_empty: opposite check
- look: fancy function to display the name, description, capacity, and items currently inside the chest

### CODE REVIEW

File: invadedhouse/rooms.py [hunk 1]
```diff
@@ -175,10 +175,52 @@ def place(self, item: Item):
         self.contents.append(item)
 
     def loot(self) -> [Item]:
+        """
+        Loots this chest (removes all items contained inside)
+
+        :return: set of Items that were in this chest (empty set if nothing was inside)
+        """
```
Comment (redjoey) [inline invadedhouse/rooms.py:182]: whats this

Comment (jsoconnell) [reply invadedhouse/rooms.py:182]: These are just comments to describe how the loot function works

File: invadedhouse/rooms.py [hunk 2]
```diff
@@ -175,10 +175,52 @@ def place(self, item: Item):
...
+        print(self.get_description())
+        print()
+
+        print(f'Holds up to {self.capacity} items.')
+        items_message = 'Currently inside: '
+        if self.is_empty():
+            items_message += 'Nothing.'
+        else:
+            items_message += ', '.join([item.get_name() for item in self.contents])
+        print(items_message)
```
Comment (redjoey) [inline invadedhouse/rooms.py:222]: cool

File: invadedhouse/rooms.py [hunk 3]
```diff
@@ -175,10 +175,52 @@ def place(self, item: Item):
...
+        if not self.contents:
+            return True
+        else:
+            return False
+
+    def is_not_empty(self):
+        if not self.contents:
+            return False
+        else:
+            return True
```
Comment (redjoey) [inline invadedhouse/rooms.py:208]: cool

**Review by redjoey — APPROVED** [review]
I like your additions, I'll aprove!

**Pull request merged**